browser/resources/settings/brave_default_extensions_page/brave_default_extensions_browser_proxy.js

```diff
@@ -21,6 +21,7 @@ cr.define('settings', function () {
     isWidevineEnabled() {}
     getRestartNeeded () {}
     wasSignInEnabledAtStartup () {}
+    wasMediaRouterEnabledAtStartup () {}
```

Comment:
same as the pref `isMediaRouterEnabled` fully captures what this value is and is consistent with c++

Reply:
updated